docs/installation.adoc

```diff
@@ -116,3 +116,38 @@ metadata:
   namespace: rhoas-operator
```

Comment:
In my mind Installation would be operator hub based installation only. Rest should move to the `build-docs` (that can be also renamed to contributing). WDYT? There is a lot to digest for possibly small utility operator we offer here.  If installation is longer than 4 steps we are not getting much aplause ;D